src/js/components/Pagination/stories/Grid.js

```diff
@@ -17,7 +24,7 @@ const CardResult = ({ item }) => (
   </Card>
 );
 
-const PaginatedGrid = () => {
+export const Grid = () => {
```

Comment:
I think we should keep the use of `Grid` to refer to the grommet component, as that will work well if cutting and pasting parts of the code and be less confusing.

Reply:
That's a good point. I'll try to figure out what exactly is the problem here.
What do you think about renaming this story as a last resort solution?

Reply:
I think we can do something similar to this: https://github.com/grommet/grommet/blob/master/src/js/components/DropButton/stories/typescript/Calendar.tsx

Reply:
Thanks Eric. This works great.